doc/ale-perl.txt

```diff
@@ -71,6 +71,53 @@ g:ale_perl_languageserver_config
   https://metacpan.org/pod/Perl::LanguageServer#Extension-Settings
 
 
+===============================================================================
```

Comment:
Where we mention the `g:` and `b:` options here the documented names need to be prefixed with `ale` like the real variable names are. The option names for Lua config don't use the prefix.